resources/js/store/modules/slides/state.js

```diff
@@ -1,6 +1,6 @@
 export default {
   slides: {
```

Comment:
change name to something that indicates the object a little bit better, something along the lines of `slideContent` or somethin

Reply:
iiiight how's `importedJSONSlides` ?